composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": "^8.1",
```

Comment:
Bumping php version in a patch release is questionable. Should there be a security issue in 1.0.3, what version number would you use for a fix?

Reply:
PHP8.0 and below are not supported anymore by PHP, so I suggest that people who are still on  7.2-8.0 upgrade as soon as possible.
I also think it's a vary small change that a security fix will be required, as wkhtmltopdf is also EOL and this library is just a wrapper for Snappy which also only supports 8.1 and above, see https://github.com/KnpLabs/snappy/blob/master/composer.json

But otherwise I guess I could release 1.0.3.1 if required?